jjb/dynamic/uperf.yml

```diff
@@ -5,19 +5,27 @@
     - shell: |+
         set -o pipefail
         set -eux
-        # Disable logging
-        set +x
-        if [[ $SKIP_TLS_VERIFICATION ]]; then
-          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL} --insecure-skip-tls-verify
-        else
-          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
-        fi
-        # Run tests
-        set -x
-        pushd workloads/network-perf
-        ./run_hostnetwork_network_test_fromgit.sh
-        ./run_pod_network_test_fromgit.sh
-        ./run_serviceip_network_test_fromgit.sh
+
+        # get perf keys to access orchestration host and set ssh session options
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
+        export OPTIONS="-o StrictHostKeyChecking=no -o ServerAliveInterval=1 -o ConnectionAttempts=100"
+        chmod 600 ${PRIVATE_KEY}
+
+        # ssh session to run tests from the orchestration host with the kubeconfig
+        ssh ${OPTIONS} -i ${PRIVATE_KEY} ${ORCHESTRATION_USER}@${ORCHESTRATION_HOST} 'bash -s'<<ENDSSH
+          set -ex
+          # Run tests
+          if [[ -d $HOME/plow ]]; then
+            rm -rf $HOME/plow
+          fi
+          git clone https://github.com/cloud-bulldozer/plow.git $HOME/plow
+          pushd $HOME/plow/workloads/network-perf/
+          ./run_hostnetwork_network_test_fromgit.sh
```

Comment:
I worry about ssh connections... and the feasibility of the scripts in this manner. Could we make a copy of the kubeconfig?

Reply:
Sure, modified it to copy the kubeconfig from the orchestration host. Thanks.